lib/analyser.js

```diff
@@ -198,84 +202,117 @@ module.exports = {
     if (showOccurrences) {
       console.log(colors.green('\n Components occurrences:'));
 
-      for (const key of Object.keys(this.occurrences)) {
-        if (!this.unusedComponents.includes(key)) {
+      Object.keys(this.components).forEach(key => {
+        let component = this.components[key];
+
+        // start with the parent components
+        if (!component.isSubComponent && component.stats.count > 0) {
           console.log(colors.cyan(`\n  ${key}:`));
 
           if (whitelist.includes(key)) {
             console.log(colors.red(`    - whitelisted`));
           }
 
-          this.occurrences[key].forEach(o => {
+          component.occurrences.forEach(o => {
             console.log(`\n   > ${o.file}`);
             o.lines.forEach(line => console.log(colors.gray(`    - ${line}`)));
           });
+
+          Object.keys(component.subComponentKeys).forEach(subComponentKey => {
+            if (this.components[subComponentKey].stats.count > 0) {
+              console.log(colors.cyan(`\n     ${subComponentKey}:`));
+              let subcomponent = this.components[subComponentKey];
+
+              subcomponent.occurrences.forEach(p => {
+                console.log(`\n       > ${p.file}`);
+                p.lines.forEach(line => console.log(colors.gray(`         - ${line}`)));
+              });
+            }
+          });
         }
-      }
+      });
     }
 
     console.log('');
   },
 
+  /**
+   * Map components based off config
```

Comment:
I'm not a native but shouldn't it be "based on config"?
```suggestion
   * Map components based on config
```

Reply:
It can be either because English is a funny language, but I like your suggestion better.